docs/hyperopt.md

```diff
@@ -150,6 +151,45 @@ The above setup expects to find ADX, RSI and Bollinger Bands in the populated in
 When you want to test an indicator that isn't used by the bot currently, remember to
 add it to the `populate_indicators()` method in `hyperopt.py`.
 
+### Using a custom loss function
+
+To use a custom loss function, make sure that the function `hyperopt_loss_custom` is defined in your custom hyperopt class.
+You then need to add the command line parameter `--loss custom` to your hyperopt call so this fuction is being used.
+
+A sample of this can be found below.
+
+``` python
+    @staticmethod
+    def hyperopt_loss_custom(results: DataFrame, trade_count: int,
+                             min_date: datetime, max_date: datetime, *args, **kwargs) -> float:
+        """
+        Objective function, returns smaller number for more optimal results
+        """
+        total_profit = results.profit_percent.sum()
+        trade_duration = results.trade_duration.mean()
+
+        trade_loss = 1 - 0.25 * exp(-(trade_count - TARGET_TRADES) ** 2 / 10 ** 5.8)
+        profit_loss = max(0, 1 - total_profit / EXPECTED_MAX_PROFIT)
+        duration_loss = 0.4 * min(trade_duration / MAX_ACCEPTED_TRADE_DURATION, 1)
+        result = trade_loss + profit_loss + duration_loss
+        return result
+```
+
+Currently, the arguments are:
+
+* `results`: DataFrame containing the result
+* `trade_count`: Amount of trades (identical to `len(results)`)
+* `min_date`: Start date of the hyperopting TimeFrame
+* `min_date`: End date of the hyperopting TimeFrame
+
+This function needs to return a floating point number (`float`). The smaller that number, the better is the result. The parameters and balancing for this are up to you.
+
+!!! Note
+    This function is called once per iteration - so please make sure to have this as optimized as possible to now slow hyperopt down unnecessarily.
+
+!!! Note
+    The last 2 arguments, `*args` and `**kwargs` are not strictly necessary but ensure compatibility for the future, so we can easily enable more parameters once we discover what could be usefull without breaking your custom hyperopt file.
```

Comment:
this note should probably be removed, it's too deep for a user, she should only follow the proposed interface for the function.